Errors while building ubuntu 14.10

Not sure what to report exactly, but I've got the following (green) error trace:

```
$ make ubuntu-14.10-box

[...]
    virtualbox-iso: Uncompressing VirtualBox 4.3.4 Guest Additions for Linux............
    virtualbox-iso: VirtualBox Guest Additions installer
    virtualbox-iso: Copying additional installer modules ...
    virtualbox-iso: Installing additional modules ...
    virtualbox-iso: Removing existing VirtualBox DKMS kernel modules ...done.
    virtualbox-iso: Removing existing VirtualBox non-DKMS kernel modules ...done.
    virtualbox-iso: Building the VirtualBox Guest Additions kernel modules
    virtualbox-iso: The headers for the current running kernel were not found. If the following
    virtualbox-iso: module compilation fails then this could be the reason.
    virtualbox-iso:
    virtualbox-iso: Building the main Guest Additions module ...done.
    virtualbox-iso: Building the shared folder support module ...fail!
    virtualbox-iso: (Look at /var/log/vboxadd-install.log to find out what went wrong)
    virtualbox-iso: Doing non-kernel setup of the Guest Additions ...done.
    virtualbox-iso: Installing the Window System drivers ...fail!
    virtualbox-iso: (Could not find the X.Org or XFree86 Window System.)
    virtualbox-iso: + true
    virtualbox-iso: + umount /mnt
    virtualbox-iso: + rm -f VBoxGuestAdditions.iso
    virtualbox-iso: + /etc/init.d/vboxadd start
    virtualbox-iso: Starting the VirtualBox Guest Additions ...fail!
    virtualbox-iso: (modprobe vboxsf failed)
    virtualbox-iso:
    virtualbox-iso:
    virtualbox-iso:
==> virtualbox-iso: Unregistering and deleting virtual machine...
==> virtualbox-iso: Deleting output directory...
Build 'virtualbox-iso' errored: Script exited with non-zero exit status: 1

==> Some builds didn't complete successfully and had errors:
--> virtualbox-iso: Script exited with non-zero exit status: 1

==> Builds finished but no artifacts were created.
make: *** [ubuntu-14.10-box] Error 1
```

Nothing was created:
```
$ make ubuntu-14.10-vagrant
cd ubuntu-14.10 && vagrant up --provision
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'ubuntu-14.10-xen' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: >= 0
==> default: Adding box 'ubuntu-14.10-xen' (v0) for provider: virtualbox
    default: Downloading: ubuntu-14.10-xen
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.

Couldn't open file /Users/thomas/git/mirage-vagrant-vms/ubuntu-14.10/ubuntu-14.10-xen
make: *** [ubuntu-14.10-vagrant] Error 1
```

I'm on OSX 10.10.3. Might be that I have a old vagrant installation, I'll try to update.

--- a/README.md
+++ b/README.md
@@ -12,9 +12,22 @@ This currently contains support for Ubuntu 14.04 LTS ("Trusty Tahr"), Ubuntu
 
 XenServer support imported from <https://github.com/jonludlam/packer-xenserver>.
 
+### Install `Virtualbox`
+
+First, install Oracle's [Virtualbox][]. Download the appropriate installer for
+your platform from <https://www.virtualbox.org/wiki/Downloads> and follow the
+instructions.
+
+[virtualbox]: https://www.virtualbox.org/
+
+Note that you need a reasonably recent version -- at time of writing, `4.3.4`
+has been observed to fail to build, while `4.3.26` has been observed to work.
+Reports of other working or failing versions welcome! (Perhaps as pull requests
+to this file in this repo :)
+
 ### Install `vagrant`
 
-First, install [Vagrant][]. On OSX I use [homebrew][] so I do this as follows:
+Next, install [Vagrant][]. On OSX I use [homebrew][] so I do this as follows:
 
     $ brew tap phinze/cask
     $ brew install brew-cask
@@ -27,8 +40,12 @@ First, install [Vagrant][]. On OSX I use [homebrew][] so I do this as follows:
 
 ### Install `packer`
 
+Then, install [Packer][]. Again, using [homebrew][] this becomes:
+
     $ brew tap homebrew/binary
     $ brew install packer
+
+[packer]: https://www.packer.io
 
 ## Use
 

